openedx/features/colaraz_features/middlewares.py

```diff
@@ -93,13 +94,16 @@ def _get_current_url(self, request):
 
         return '{}://{}{}'.format(schema, domain, request.path)
         
-    def _get_user_site_domain(self, user, session):
+    def _get_user_site_identifier(self, user, session):
         """
         Returns domain of site associated with the User.
         """
-        if not session.get('user_site_domain', None):
-            session['user_site_domain'] = UserAttribute.get_user_attribute(user, 'created_on_site')
-        return session.get('user_site_domain', None)
+        is_new_user = False
+        if not session.get('user_site_identifier', None):
```

Comment:
```suggestion
        if not session.get('user_site_identifier'):
```

Reply:
just trying to stay consistent with open edX.